src/fu-engine.c

```diff
@@ -52,13 +52,15 @@
 #include "fu-security-attrs-private.h"
 #include "fu-smbios-private.h"
 #include "fu-udev-device-private.h"
+#include "fu-ble-device.h"
 
 #ifdef HAVE_GUDEV
 #include "fu-udev-backend.h"
 #endif
 #ifdef HAVE_GUSB
 #include "fu-usb-backend.h"
 #endif
+#include "fu-bluez-backend.h"
```

Comment:
I think we actually want to have the bluez backend as optional.  We won't have bluez in Windows for example.

Reply:
I'm splitting this up right now :)

Reply:
So, we don't actually *need* to make it optional; everything is done over DBus. For win32 we'll just fail to coldplug, which might be fine. Maybe we don't want to compile it at all, opinions welcome.

Reply:
Well aside from Win32, what about BSD, ChromeOS?  Do they use bluez?  I'm wondering if we want to make it clearer with a meson option `-Dbluetooth=true` and only compile plugins that use bluez when that's set.

Reply:
We could certainly add -Dbluez but I think it needs to be conditional on the implementation, not the abstract functionality. I think if an alternate bluetooth implementation provides enumeration, and a UUID read/write I think devices would work just as well.

Reply:
OK, I think in that case `-Dbluez` is a good idea.

Reply:
Done